ush/python/pygfs/task/aero_analysis.py

```diff
@@ -200,13 +200,16 @@ def clean(self):
     @logit(logger)
     def _add_fms_cube_sphere_increments(self: Analysis) -> None:
         """This method adds increments to RESTART files to get an analysis
-        NOTE this is only needed for now because the model cannot read aerosol increments.
-        This method will be assumed to be deprecated before this is implemented operationally
         """
+        if self.task_config.DOIAU:
+            restarttime = self.task_config.AERO_WINDOW_BEGIN
+        else:
+            restarttime = self.task_config.current_cycle
```

Comment:
If this is elevated to `__init__` as in the previous comment, we won't need to keep calculating it.